### ISSUE TITLE
Order Approval: It is possible to create the same rule multiple times at once

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#4356]
When clicking the save button on the approval rule detail page, it is possible to create the rule several times - each time for every click.

*Steps to reproduce:*

# Create a new rule
# Click on the save button multiple times, before you are being navigated back to the rule listing

*Expected result:*

The rule has been created only once

*Actual result:*

The rule is created multiple times

**Comment by shopwareBot** [comment]
[public] Automated response: This issue is linked to the internal issue [https://shopware.atlassian.net/browse/NEXT-36406](https://shopware.atlassian.net/browse/NEXT-36406).

 

[created from NEXT-36406, comment 486607]

**Comment by shopwareBot** [comment]
[public] Hello,

 Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.

 If you feel like this issue is still impacting you, please create a new issue and let us know. Thank you again for your valuable feedback.



[created from NEXT-36406, comment 532561]
